Header header
float64 x_min
float64 x_max
float64 y_min
float64 y_max
float64 radius
float64 resolution
float64 rows
float64 columns
CLiFFMapLocationMsg[] locations